### ISSUE TITLE
About torch.autograd.Function and Custom Backpropagation in PyTorch

### ISSUE THREAD

**Issue by qulunting** [NVlabs/pacnet#32]
I implemented a 1D convolution using pixel-adaptive convolution, but while ensuring the forward propagation is correct, the gradients of the weights returned by PyTorch's automatic backpropagation are incorrect.Why does the blogger use torch.autograd.Function to customize backpropagation instead of PyTorch's automatic backpropagation? Why are the gradients of the weights returned by PyTorch's automatic backpropagation incorrect?
